Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM node:7
+MAINTAINER Andrew Krug <akrug@mozilla.com>
+
+RUN mkdir /nlx
+WORKDIR /nlx
+
+# Setup the container for caching the node modules in case of npm repository failures.
+ADD package.json package.json
+ADD package-lock.json package-lock.json
+ADD ci/ ci
+RUN chmod +x ci/*/*.sh
+
+# Get the container OS up to date
```

Comment:
total nit but id move that up to speed up rebuilds , since you probably want to invalidate `ADD ci/ ci` instead

something like that:
```
RUN apt-get update....

...

ARG CACHE=1
ADD ci/ ci
```

then you can have a Makefile target for ex:

```
build-nocache:
   docker build CACHE=$(shell date +%s) # faster update when ci/* changes

build:
   docker build --nocache #forces full update so it grabs OS updates
```